polls/tests.py

```diff
@@ -1,3 +1,63 @@
+from django.contrib.auth import get_user_model
+from django.urls import reverse
+from django.utils import timezone
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from .models import Annotation, Batch, Task
+
 # from django.test import TestCase
 
-# Create your tests here.
+
+class LockUserAnnotationListTest(APITestCase):
+    """This class sets up the test db and tests
+    - 'lock-users-api' endpoint
+    - 'annotation-api' endpoint
+    """
+
+    def setUp(self):
+        self.user = get_user_model().objects.create(
+            username="test_user", password="testpass"
+        )
```

Comment:
We can do that `user=get_user_model()` and then we can use later everywhere. It is good practice.

Reply:
This is a test setUp and it's class, defining a variable under a class requires the use of the self keyword to show that the variable belongs to the instance of the class.

Reply:
@certaintydev2 I think I agree with @DeFidelity here